WORKSPACE

```diff
@@ -14,170 +14,30 @@
 
 workspace(name = "io_opencensus_cpp")
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load(":deps.bzl", "opencensus_cpp_deps")
```

Comment:
IMO we should put deps.bzl into the bazel/ directory to match what other projects (grpc, upb, envoy, google-cloud-cpp) do.